src/gc/gcconfig.cpp

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "common.h"
+#include "gcenv.h"
+#include "gc.h"
+
+#define BOOL_CONFIG(name, key, default, unused_doc)            \
+  bool GCConfig::Get##name()                                   \
```

Comment:
so why do we even need to check for cached values? We are not just getting all the configs on startup?